test/unit/utils.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2014, 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A collection of functions that helps running unit tests"""
+
+import functools
+import re
+import unittest
+
+import nose.plugins.skip
+
+
+def skipIf(condition, reason):
+    """A Python 2.6 skipIf decorator."""
```

Comment:
Since we always use this (also on Python >= 2.7), maybe the docstring could use some tweaking:
```A `skipIf` decorator

Similar to `unittest.skipIf`, for Python 2.6 compatibility.
```